solarnet/README.md

```diff
@@ -93,4 +123,73 @@ one-liners be automated.
 
 ### unsupported parameter for module: restart_policy
 
-You're probably using the system version of Ansible, and it is outdated (< 1.9). Make sure to follow the Ansible setup steps above, and load the virtualenv. This will load a working version of Ansible.
+You're probably using the system version of Ansible, and it is outdated (< 1.9).
+Make sure to follow the Ansible setup steps above, and load the virtualenv.
+This will load a working version of Ansible.
+
+## Monitoring
+
+We use Prometheus to scrape and store timeseries
+from IPFS and the hosts themselves.
+Grafana provides the dashboard UI.
+
+Both are available at http://metrics.i.ipfs.io
+http://metrics.i.ipfs.io/prometheus, respectively.
+
+- You need to be peered into the Hyperboria cjdns network
+  in order to reach the address that this domain name points to.
+- Your cjdns address needs to be whitelisted.
+
+### Cjdns
+
+In addition to serving http://h.gateway.ipfs.io,
+we use cjdns for a very simple VPN based on an IP address whitelist.
+
+```sh
+$ git clone https://github.com/hyperboria/cjdns.git
+$ cd cjdns/
+$ ./do
+$ ./cjdroute --genconf > cjdroute.conf
+```
+
+This creates a `tunX` network interface,
+which grabs all `fc00::/8` traffic and hands it to cjdns.
+
+Scripts for various init systems are provided in `contrib/`.
+
+### Peering
+
+Cjdns nodes peer automatically on local networks,
+but on WANs like the internet, peering requires credentials.
+You can peer with the Solarnet cjdns nodes
+by using one of the `cjdns_authorized_passwords` in `secrets.yml`.
+
+Note that peering with any other node in the Hyperboria network is sufficient, too.
+
+You can check the status of your peerings:
+
+```sh
+$ watch tools/peerStats
+```
+
+Or check connectivity to solarnet:
+
+```sh
+$ ping6 h.gateway.ipfs.io
+```
+
+### Authentication
+
+Since IP packets in cjdns are authenticated and encrypted,
+we can use the HTTP client's IP address for authentication,
+instead of Basic Auth or another login mechanism.
```

Comment:
@lgierth yeah, what i mean is that the local state of stream ciphers advance as you use them (i.e. they're chained), they assume a reliable stream because (if a packet is lost and not recovered) the otherside may miss packets. `TLS / TCP / IP` vs `DTLS / UDP / IP`. I would expect cjdns to use either a layer of reliability + TLS style cipher, OR something like DTLS over the unreliable transport.